modules/RTC/TraceablePeerConnection.js

```diff
@@ -1035,6 +1035,33 @@ const normalizePlanB = function(desc) {
 };
 
 /**
+ * Makes sure that both audio and video directions are configured as 'sendrecv'.
+ * @param {Object} localDescription the SDP object as defined by WebRTC.
+ */
+const enforceSendRecv = function(localDescription) {
```

Comment:
so if there are no ssrcs in the mline, will chrome complain about setting it to sendrecv?